### PR TITLE
Np 47839 tests use real network calls

### PR DESCRIPTION
**Pull request by torarnet**
Fixed some tests that did real network calls against our environments.

### CODE REVIEW

File: expansion/src/main/java/no/unit/nva/expansion/ResourceExpansionServiceImpl.java [hunk 1]
```diff
@@ -55,14 +54,6 @@ public class ResourceExpansionServiceImpl implements ResourceExpansionService {
     private final RawContentRetriever authorizedUriRetriever;
     private final RawContentRetriever uriRetriever;
 
-    public ResourceExpansionServiceImpl(ResourceService resourceService,
-                                        TicketService ticketService) {
-        this.resourceService = resourceService;
-        this.ticketService = ticketService;
-        this.authorizedUriRetriever = new UriRetriever();
```
Comment (torarnet) [inline expansion/src/main/java/no/unit/nva/expansion/ResourceExpansionServiceImpl.java:62]: Removing redundant constructor causing issues with coverage, while avoiding implicit generation of an UriRetriever that we cannot mock. The other constructor forces us to supply it as parameter.

File: expansion/src/test/java/no/unit/nva/expansion/model/ExpandedDataEntryTest.java [hunk 2]
```diff
@@ -146,7 +147,6 @@ void shouldExpandImportCandidateSuccessfully(PublicationContext publicationConte
 
         var expandedImportCandidate = ExpandedImportCandidate.fromImportCandidate(importCandidate, uriRetriever);
         assertThat(importCandidate.getIdentifier(), is(equalTo(expandedImportCandidate.identifyExpandedEntry())));
-        this.resourceExpansionService = new ResourceExpansionServiceImpl(resourceService, ticketService);
```
Comment (torarnet) [inline expansion/src/test/java/no/unit/nva/expansion/model/ExpandedDataEntryTest.java:149]: Did not seem to be in use

File: publication-event-handlers/src/test/java/no/unit/nva/publication/events/handlers/persistence/AnalyticsIntegrationHandlerTest.java [hunk 3]
```diff
@@ -73,16 +72,13 @@ private static List<Class<?>> listPublicationInstanceTypes() {
     @BeforeEach()
     public void init() {
         super.init();
-        this.dynamoClient = super.client;
```
Comment (torarnet) [inline publication-event-handlers/src/test/java/no/unit/nva/publication/events/handlers/persistence/AnalyticsIntegrationHandlerTest.java:76]: Did not seem to be in use

File: publication-event-handlers/src/test/java/no/unit/nva/publication/events/handlers/persistence/AnalyticsIntegrationHandlerTest.java [hunk 4]
```diff
@@ -73,16 +72,13 @@ private static List<Class<?>> listPublicationInstanceTypes() {
...
         this.outputStream = new ByteArrayOutputStream();
-        FakeS3Client s3Client = new FakeS3Client();
+        S3Client s3Client = new FakeS3Client();
         this.analyticsIntegration = new AnalyticsIntegrationHandler(s3Client);
         this.s3Driver = new S3Driver(s3Client, "notImportant");
 
         resourceService = getResourceServiceBuilder().build();
         ticketService = getTicketService();
-
-        this.resourceExpansionService = setupResourceExpansionService();
```
Comment (torarnet) [inline publication-event-handlers/src/test/java/no/unit/nva/publication/events/handlers/persistence/AnalyticsIntegrationHandlerTest.java:85]: Moved to the single test that actually used it

**Review by brinxmat — APPROVED** [review]

**Review by truhacevkir — APPROVED** [review]

**Review by torbjokv — APPROVED** [review]

**Pull request merged**